presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -84,6 +84,8 @@ class SystemConfig : public ConfigBase {
   static constexpr std::string_view kHttpExecThreads{"http_exec_threads"};
   static constexpr std::string_view kNumIoThreads{"num-io-threads"};
   static constexpr std::string_view kNumSpillThreads{"num-spill-threads"};
+  static constexpr std::string_view kSpillerSpillPath =
```

Comment:
Is it the path we used in staging test? Thanks!

Reply:
> Is it the path we used in staging test? Thanks!

@xiaoxmeng 
What is "staging test"?